### ISSUE TITLE
Publication order is a bit confusing

### ISSUE THREAD

**Issue by ssfrr** [ResEnv/resenv-site#6]
I really like the aesthetic of the paper list, but the arrangement has the publications going all the way down to the bottom of the page in the left column before jumping up to the right. This is a bit hard to follow as you have to jump all the way back to the top while you're reading through the papers. Also at first glance it make the papers at the top of the right column look like they're part of the newest block.

Perhaps an arrangement more like the following:

```
2014
----

Paper 1    Paper 3
Paper 2    Paper 4

2013
----

Paper 5    Paper 7
Paper 6    Paper 8
```

I think this could keep the general look the same but be easier to navigate.

**Comment by d0nd3r3k** [comment]
Good point, I was taken by the fact that it looks and feels like a 'research-paper' but it's true, research papers layout is hard to follow, what you're suggesting keeps the looks without sacrificing the feel!

**Comment by ssfrr** [comment]
Research papers are also only 1-page long before you loop up to the 2nd column. This would be more like going through all the pages reading the left column, then going back to the first page to start the right column. :)

**Comment by d0nd3r3k** [comment]
Well said, now I'm totally sold :p

**Comment by d0nd3r3k** [comment]
I really don't like the publication page, let's revert to the old 2-col design and re-do as discussed here!
